### ISSUE TITLE
genesis update should not require prerequisite checks

### ISSUE THREAD

**Issue by norman-abramovitz** [genesis-community/genesis#513]
genesis prerequisites should not be required for genesis update

genesis update -c -p
GENESIS PRE-REQUISITES CHECKS FAILED!!

Encountered the following errors:
  - Missing `credhub` -- install from https://github.com/cloudfoundry-incubator/credhub-cli/releases

**Comment by dennisjbell** [comment]
Agreed, but it should print them out AFTER upgrade.  The rub is of course we can only fix this behaviour in a future version of genesis, so the one that initiates the upgrade that you already have will still have this problem, so you'll still have to install the missing dependencies (or just pull down a new version from github, and then still have to install the missing dependencies).
